dml/token.go

```diff
@@ -5,28 +5,24 @@ import (
 	"unicode/utf8"
 )
 
-// identifier: \p{L}+[_\p{L}0-9]* with support for matched quotes
+// identifier: \p{L}+[\p{L}0-9_-]*[\p{L}0-9]+$ with support for matched quotes
```

Comment:
Maybe a dumb question, but wouldn't this be simpler for interpretation? Not sure if the validated content changes

```suggestion
// identifier: \p{L}+[\p{L}0-9_-]*[\p{L}0-9]?$ with support for matched quotes
```